src/math/sat.rs

```diff
@@ -0,0 +1,34 @@
+use cgmath::{BaseFloat, EuclideanSpace, Point3, Vector3};
```

Comment:
found the missing file of the nit

Reply:
You were right, it was missing. I pushed it.